R/get_sections.R

```diff
@@ -67,81 +73,83 @@
 #' @param econ_class \code{character}. Filter sections to those containing a
 #'   named economic attribute class (e.g., "material", "water", "precious
 #'   commodity").
-#' @param pbdb_collection_no \code{integer}. Filter sections to those containing
-#'   one or more Paleobiology Database collection(s) as specified by their
-#'   unique identification number(s).
-#' @param project_id \code{integer}. Filter sections to those contained within a
-#'   Macrostrat project as specified by its unique identification number.
+#' @param project_id \code{integer}. Filter sections to those contained within
+#'   one or more Macrostrat project(s) as specified by their unique
+#'   identification number(s).
 #' @param adjacents \code{logical}. If `column_id` or `lat`/`lng` is specified,
 #'   should all sections that touch the specified column be returned? Defaults
 #'   to `FALSE`.
+#'
 #' @return A \code{dataframe} containing the following columns:
 #' \itemize{
-#'   \item \code{col_id}: The unique Macrostrat column identification number.
+#'   \item \code{col_id}: The unique identification number of the Macrostrat
+#'     column.
```

Comment:
Maybe change to "The unique identification number of the Macrostrat column containing the section" just to make the distinction between these super clear?